src/app.js

```diff
@@ -18,16 +18,11 @@ const getItemsForNames = (menuItemNames) => (
  * @returns {string} the formatted, localised price
  */
 const localisePrice = (price, locale) => {
-  /*
-   * Implement me!
-   * Hint: .toFixed() might help...
-   */
-   if (locale === 'de') {
-     return price.toFixed(2).replace('.', ',') + ' €';
-   } else {
-     return '€' + price.toFixed(2);
-
-   }
+  if (locale === "de") {
+    return price.toFixed(2).replace(".", ",") + " €";
+  } else {
+    return "€" + price.toFixed(2);
+  }
```

Comment:
Here's a possible refactoring:
```
const priceString = price.toFixed(2);
return locale === 'de' ? priceString.replace('.', ',') + ' €' : '€' + priceString;
```
Assigning the `priceString` variable avoids repeating the `toFixed(2)`. This follows the DRY principle.